### ISSUE TITLE
Swin V2 Fine-tuning a Fine-tuned Checkpoint

### ISSUE THREAD

**Issue by collinmccarthy** [microsoft/Swin-Transformer#259]
Hello,

Let's say I want to fine-tune the [swinv2_large_patch4_window12to24_192to384_22kto1k_ft](https://github.com/microsoft/Swin-Transformer/blob/main/configs/swinv2/swinv2_large_patch4_window12to24_192to384_22kto1k_ft.yaml) pre-trained checkpoint for a new task/resolution. It was already fine-tuned on ImageNet-1k using `PRETRAINED_WINDOW_SIZES: [ 12, 12, 12, 6 ]` which makes sense given the feature map height/width is [48, 24, 12, 6] at the start of the four stages for an input of 192x192 and a patch size of 4. 

But now that I'm fine tuning again should I update this to `PRETRAINED_WINDOW_SIZES: [ 24, 24, 24, 12]` (given feature map height/width [96, 48, 24, 12]) since that's what this checkpoint was fine-tuned with, or leave it where it was before? And I assume whatever the answer is, I should keep it at that value even if resuming from a latest checkpoint?

Thanks,
-Collin

**Comment by collinmccarthy** [comment]
It seems to make the most sense to me to change it to `PRETRAINED_WINDOW_SIZES: [ 24, 24, 24, 12]` during the new fine-tuning run, and leave it there until the model converged (and keep it set to that for inference as well, obviously). 

Would you please confirm this is correct? Thank you

**Comment by collinmccarthy** [comment]
I tried both settings and PRETRAINED_WINDOW_SIZES: [ 0, 0, 0, 0] and found no difference in the model accuracy when fine-tuning on cityscapes. My assumption is this only matters for inference when the window size during inference is different from the window sized used during training.
